spec/models/story_parser_spec.rb

```diff
@@ -88,11 +88,10 @@
         expect(@sp.get_source_if_known(StoryParser::KNOWN_STORY_LOCATIONS, url)).to eq("lj")
       end
 
-      # TODO: uncomment and remove this comment when (if) fixing the bug
-      it "should match a folder style link to an individual user on journalfen" # do
-      #   url = "http://www.journalfen.net/users/username"
-      #   @sp.get_source_if_known(StoryParser::KNOWN_STORY_LOCATIONS, url).should eq("lj")
-      # end
+      it "should match a folder style link to an individual user on journalfen"  do
+         url = "http://www.journalfen.net/users/username"
+         expect(@sp.get_source_if_known(StoryParser::KNOWN_STORY_LOCATIONS, url)).to eq("lj")
+       end
```

Comment:
`end` at 94, 7 is not aligned with `it "should match a folder style link to an individual user on journalfen"  do` at 91, 6